apps/web/package.json

```diff
@@ -10,6 +10,7 @@
     "@gallery-so/visitor-plugin-common": "^0.0.2",
     "@graphql-codegen/cli": "^2.11.6",
     "@graphql-codegen/typescript": "^2.7.3",
+    "@moonrepo/cli": "^0.23.4",
```

Comment:
I want to look into a way to make sure that we don't let each app fall out of sync with it's siblings' / parent's dependencies.

Syncpack seems interesting. Will look into this some more